roofit/roofitcore/inc/RooFit/Detail/EvaluateFuncs.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Project: RooFit
+ * Authors:
+ *   Jonas Rembser, CERN 2023
+ *   Garima Singh, CERN 2023
+ *
+ * Copyright (c) 2023, CERN
+ *
+ * Redistribution and use in source and binary forms,
+ * with or without modification, are permitted according to the terms
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
+ */
+
+#ifndef RooFit_Detail_EvaluateFuncs_h
+#define RooFit_Detail_EvaluateFuncs_h
+
+#include <cmath>
+
+namespace RooFit {
+
+namespace Detail {
+
+namespace EvaluateFuncs {
+
+/// @brief Function to evaluate an un-normalized RooGaussian.
```

Comment:
To avoid code duplication, you should use this `gaussEvaluate` function also in `RooGaussian::evaluate()` inside RooGaussian.cxx. Better to do this now before we forget.